learningresources/migrations/0015_backfill_curator_vocabularies.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.utils.text import slugify
+
+from taxonomy.signals import create_default_vocabulary
+
+# pylint: skip-file
+
+VOCAB_NAME = "curation status"
+EMPTY_VALUE = "--not set--"
+
+
+def backfill_curator_vocabs(apps, schema_editor):
+    """Backfill auto-generated vocabulary to existing repos."""
+    Repository = apps.get_model("learningresources", "Repository")
+    LearningResourceType = apps.get_model(
+        "learningresources", "LearningResourceType")
+    Vocabulary = apps.get_model("taxonomy", "Vocabulary")
+    learning_resource_types = LearningResourceType.objects.all()
+    for repo in Repository.objects.all().iterator():
+        if Vocabulary.objects.filter(
+            name=VOCAB_NAME, repository__id=repo.id).exists():
+            continue
+        vocab = Vocabulary.objects.create(
+            repository_id=repo.id,
+            name=VOCAB_NAME, description=VOCAB_NAME,
+            required=True, vocabulary_type='m',
+            weight=0, multi_terms=False,
+            slug=slugify("{0}_{1}_backfill".format(VOCAB_NAME, repo.id)),
+        )
+        for resource_type in learning_resource_types:
+            vocab.learning_resource_types.add(resource_type)
+        create_terms(apps, vocab.id)
+        backfill_resources(apps, repo.id)
+
+
+def create_terms(apps, vocab_id):
+    """Create the curation status terms for a newly-created Vocabulary."""
+    Term = apps.get_model("taxonomy", "Term")
+    labels = (
+        'ready to use', 'tagged', 'course information',
+        'discarded', 'hidden', EMPTY_VALUE,
+    )
```

Comment:
Yes, you're right. The `create_default_vocabulary` function in taxonomy signals should also create the terms. Good catch, thank you.